packages/uni_ui/lib/cards/library_occupation_card.dart

```diff
@@ -53,25 +53,17 @@ class FloorOccupationWidget extends StatelessWidget {
 class LibraryOccupationCard extends StatelessWidget {
   const LibraryOccupationCard({
     super.key,
-    required this.occupationMap,
-    required this.floorText,
+    required this.capacity,
+    required this.occupation,
+    required this.occupationCardsList,
   });
 
-  final Map<String, dynamic> occupationMap;
-  final String floorText;
+  final int capacity;
+  final int occupation;
+  final List<Widget> occupationCardsList;
```

Comment:
Because you already know the type of widget to receive, you can write `FloorOccupationWidget` directly